worldedit-bukkit/src/main/java/com/fastasyncworldedit/bukkit/regions/plotsquared/PlotSquaredFeature.java

```diff
@@ -174,22 +173,21 @@ public FaweMask getMask(Player player, MaskType type, boolean isWhitelist, boole
             return null;
         }
 
+        final World world = player.getWorld();
+        int min = area != null ? area.getMinBuildHeight() : world.getMinY();
+        // PlotSquared uses exclusive max height, WorldEdit uses inclusive max height -> subtract 1
+        int max = area != null ? Math.min(world.getMaxY(), area.getMaxBuildHeight() - 1) : world.getMaxY();
         Region maskedRegion;
         if (regions.size() == 1) {
-            final World world = player.getWorld();
-            int min = area != null ? area.getMinBuildHeight() : world.getMinY();
-            // PlotSquared uses exclusive max height, WorldEdit uses inclusive max height -> subtract 1
-            int max = area != null ? Math.min(world.getMaxY(), area.getMaxBuildHeight() - 1) : world.getMaxY();
 
             final CuboidRegion region = regions.iterator().next();
             final BlockVector3 pos1 = BlockVector3.at(region.getMinimumX(), min, region.getMinimumZ());
             final BlockVector3 pos2 = BlockVector3.at(region.getMaximumX(), max, region.getMaximumZ());
             maskedRegion = new CuboidRegion(pos1, pos2);
         } else {
-            World world = FaweAPI.getWorld(area.getWorldName());
```

Comment:
Is there any specific reason why the world was obtained by the area directly, instead of the players current world? (As the world variable was combined to just the users world in L176)

Reply:
I couldn't see any reason for that tbh, possibly left over from a long time ago?